.github/workflows/go_tests.yml

```diff
@@ -65,3 +66,8 @@ jobs:
           go install "honnef.co/go/tools/cmd/staticcheck@2023.1.3"
           cd sdks/go/pkg/beam
           $(go env GOPATH)/bin/staticcheck ./...
+      - name: Run govulncheck
+        run: |
+          go install golang.org/x/vuln/cmd/govulncheck@latest
+          cd sdks/go/pkg/beam
```

Comment:
Does this fail if set at sdks/go? We are also concerned with vulnerabilities in the container boot loaders, and this isn't covered.

Ideally we also check 'sdks/java/container/*', 'sdks/python/container/*', 'sdks/typescript/container/*'

Reply:
It does not fail, we can still cover bootloader tooling. I can change that. We could also invoke at the sdk/ directory and potentially cover the entire code base here